ios/FluentUI/Vnext/Notification/NotificationView_SwiftUI.swift

```diff
@@ -161,9 +164,9 @@ public struct NotificationViewSwiftUI: View {
     }
 
     public var body: some View {
-        // Note: we are using viewController's width because GeometryReader relies on the bounds of the hosting controller which cannot be set accurately at this point
-        if let windowWidth = windowProvider?.window?.bounds.width {
-            let width = windowWidth - safeAreaInsets.left - safeAreaInsets.right
+        // Note: we are using keyWindow's width because GeometryReader relies on the bounds of the hosting controller which cannot be set accurately at this point
+        if let windowWidth = UIApplication.shared.keyWindow?.bounds.width {
```

Comment:
Note: I know we spent time trying to move away from this, but since MSFNotification conforms to ControlHostingContainer now, the windowProvider can't be set (error: 'self' used before 'super.init' call). The pure SwiftUI implementation I'll be working on after this PR will be replacing this with GeometryReader, so it's just a temporary fix.